src/Compilers/CSharp/Test/Semantic/Semantics/StackAllocSpanExpressionsTests.cs

```diff
@@ -742,5 +742,57 @@ void M()
                 //         var implicitError = explicitError.Length > 0 ? stackalloc NonComparable[10] : stackalloc NonComparable[100];
                 Diagnostic(ErrorCode.ERR_GenericConstraintNotSatisfiedValType, "NonComparable[100]").WithArguments("System.Span<T>", "System.IComparable", "T", "NonComparable").WithLocation(20, 98));
         }
+
+        [Fact]
+        [WorkItem(26195, "https://github.com/dotnet/roslyn/issues/26195")]
+        public void StackAllocImplicitConversion_TwpStep_ToPointer()
+        {
+            var code = @"
+class Test2
+{
+}
+unsafe class Test
+{
+	public void Method()
+	{
+		Test obj1 = stackalloc int[2];
+	}
+	public static implicit operator Test2(int* value) => default;
```

Comment:
>Test2 [](start = 33, length = 5)

Do we have tests for scenarios when this is "Test" instead of "Test2"?

Reply:
Yes. Please check the tests at the start of this file for semantic model, and tests at the end of `CodeGenTests.cs` for emitting the conversion.

---
In reply to: [183469924](https://github.com/dotnet/roslyn/pull/26248#discussion_r183469924) [](ancestors = 183469924)